### PR TITLE
Prepare for v1.2.0

### LINKED ISSUE #393: Release v1.2.0 of Ceph-CSI driver
- [x] Define release process
  - Release doc would be a good start
- [x] Define release deadlines
  - Code Freeze : 05-Aug-2019
  - RC : 10-Aug -2019
  - GA : **15-Aug-2019**
  - GA postponed to **30-Aug-2019**  
- [x] Define/list roadmap for v1.2.0
  - Proposed contents for v1.2.0 release
  -- Metrics support ( RBD, CephFS)
  -- Resize support for Ceph and RBD plugin  **[Deferred]**
  -- Snapshot support for RBD.**[Deferred]**
  -- Cloning support for RBD.**[Deferred]**
- [x] Release Team:
 - @Madhu-1 
 - @humblec 
 
  - The release blockers (issues/PRs) are labelled with `release-1.2.0`.
https://github.com/ceph/ceph-csi/labels/release-1.2.0

- [x] Document features and announce the release.

### PR DESCRIPTION
**Pull request by Madhu-1**
Fixes #393
update images and readme for v.1.2.0 branch

adding DNM label till we get cephfs  patch  in

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 we can add #393 issue in PR description

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@
 CONTAINER_CMD?=docker
 
 CSI_IMAGE_NAME=$(if $(ENV_CSI_IMAGE_NAME),$(ENV_CSI_IMAGE_NAME),quay.io/cephcsi/cephcsi)
-CSI_IMAGE_VERSION=$(if $(ENV_CSI_IMAGE_VERSION),$(ENV_CSI_IMAGE_VERSION),canary)
```
Comment (ShyamsundarR) [inline Makefile]: Is the intention to change this back to 1.2-canary post the release? I am asking, as post 1.2.0 if any PR is merged to the 1.2.0 branch, we would update the same image in quay (afaict), which is not desirable.

File: deploy.sh [hunk 2]
```diff
@@ -24,11 +24,8 @@ push_helm_charts() {
 	fi
 }
 
-if [ "${TRAVIS_BRANCH}" == 'csi-v0.3' ]; then
-	export ENV_RBD_IMAGE_VERSION='v0.3-canary'
-	export ENV_CEPHFS_IMAGE_VERSION='v0.3-canary'
-elif [ "${TRAVIS_BRANCH}" == 'master' ]; then
-	export ENV_CSI_IMAGE_VERSION='canary'
+if [ "${TRAVIS_BRANCH}" == 'release-v1.2.0' ]; then
+	export ENV_CSI_IMAGE_VERSION='v1.2.0'
```
Comment (ShyamsundarR) [inline deploy.sh]: Same comment as the one in Makefile here as well.

File: Makefile (hunk 1 above)
Comment (Madhu-1) [reply Makefile]: yes will revert it back to 1.2-canary

File: deploy.sh (hunk 2 above)
Comment (Madhu-1) [reply deploy.sh]: yes will revert it back to 1.2-canary

**Comment by Madhu-1** [comment]
update helm values

**Comment by ShyamsundarR** [comment]
Looks good, reserving the approval to wait for required patches and overall branch readiness.

**Comment by humblec** [comment]
Merging this manually as we are really running out of time and rook release is waiting on us.

**Comment by Madhu-1** [comment]
@humblec rebased the PR. PTAL

**Comment by humblec** [comment]
LGTM. Approved .

**Pull request merged**